src/Core__Nullable.res

```diff
@@ -13,3 +13,35 @@ let fromOption: option<'a> => t<'a> = option =>
   | Some(x) => make(x)
   | None => undefined
   }
+
+let getWithDefault = (value, default) =>
+  switch value->toOption {
+  | Some(x) => x
+  | None => default
+  }
+
+let getExn: t<'a> => 'a = value =>
+  switch value->toOption {
+  | Some(x) => x
+  | None => raise(Invalid_argument("Nullable.getExn: value is null or undefined"))
```

Comment:
In other modules, this raises `Not_found`. I think a more specific error message is better though, so I'd rather change the others to this format.

Or better yet, for exceptions that don't make sense to catch, I tend to use a `panic` function that raises a proper JS error, which yields a much better debugging experience.

Reply:
I agree a specific error message is more helpful.

`panic` sounds interesting, where do you envision that living? In `Error`?

Reply:
I have it in the global scope, which would be directly in `Core` in this case. But it's really just an alias for `Exn.raiseError`, just named so that its intent is made clear. Could just use that instead too.

Reply:
@cknitt what do you think? I think having it in the global scope sounds quite interesting, but curious what others think.

Reply:
I am not seeing myself using that much, but I wouldn't mind having a `panic` function available globally.

Reply:
Cool. I'll add it in a separate PR.